### PR TITLE
parallelize the user fetches, this is a nice 3x speedup

### PR DESCRIPTION
**Pull request by maxtaco**
- but i think better to just do it with one RTT

### CODE REVIEW

**Comment by maxtaco** [comment]
likely a throwaway, but post it nonetheless

File: go/teams/service_helper.go [hunk 1]
```diff
@@ -122,14 +122,40 @@ func userVersionToDetails(ctx context.Context, g *libkb.GlobalContext, uv keybas
...
+	}
+	retCh := make(chan res, len(uvs))
+	width := 10
+	gate := make(chan struct{}, width)
+	for i, uv := range uvs {
+		go func(i int, uv keybase1.UserVersion) {
+
+			// This will block when there are more than `width` number of goroutines
+			// outstanding, and will unblock as soon as there is room.
+			gate <- struct{}{}
```
Comment (zapu) [inline go/teams/service_helper.go]: list.go used `semaphore` for this: https://github.com/keybase/client/blob/master/go/teams/list.go#L82-L87

do we prefer one more than the other or it doesn't matter?

Comment (maxtaco) [reply go/teams/service_helper.go]: i remember that, this seemed easier though, or does it seem too hacky?

Comment (zapu) [reply go/teams/service_helper.go]: I'd say looks exactly how one would expect go code to look like :)

Comment (maxtaco) [reply go/teams/service_helper.go]: Can actually swap 135 and 139

Comment (maxtaco) [reply go/teams/service_helper.go]: Anyone else have opinions? I think mine is easier to understand but it doesn’t really matter.

**Comment by keybase-ci-visdiff** [comment]
:no_entry: Error rendering the commits ccadec96c565...f84fce3eda1e on linux. <details><summary>:mag_right: 23 errored, 7 new, 40 changed</summary>

 * Error in f84fce3eda1e: **Chat_No_Conversation-normal-ERROR**  
   Chat_No_Conversation-normal-ERROR rendered
 * Error in f84fce3eda1e: **Chat_Inbox-Selected_Normal-ERROR**  
   Chat_Inbox-Selected_Normal-ERROR rendered
 * Error in f84fce3eda1e: **Chat_Inbox-Selected_Muted-ERROR**  
   Chat_Inbox-Selected_Muted-ERROR rendered
 * Error in f84fce3eda1e: **Chat_Inbox-Normal-ERROR**  
   Chat_Inbox-Normal-ERROR rendered
 * Error in ccadec96c565: **You_Rekey-Normal-ERROR**  
   You_Rekey-Normal-ERROR rendered
 * Error in ccadec96c565: **Devices_Devices_List-Devices-ERROR**  
   Devices_Devices_List-Devices-ERROR rendered
 * Error in f84fce3eda1e: **Devices_Devices_List-Devices-ERROR**  
   Devices_Devices_List-Devices-ERROR rendered
 * Error in ccadec96c565: **Chat_Input-Normal-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Chat_Input-Normal-ERROR-ccadec96c565.png)
 * Error in f84fce3eda1e: **Chat_Input-Normal-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Chat_Input-Normal-ERROR-f84fce3eda1e.png)
 * Error in ccadec96c565: **Chat_Inbox-You_Rekey_Selected-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Chat_Inbox-You_Rekey_Selected-ERROR-ccadec96c565.png)
 * Error in f84fce3eda1e: **Chat_Inbox-You_Rekey_Selected-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Chat_Inbox-You_Rekey_Selected-ERROR-f84fce3eda1e.png)
 * Error in ccadec96c565: **Chat_Inbox-You_Rekey-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Chat_Inbox-You_Rekey-ERROR-ccadec96c565.png)
 * Error in f84fce3eda1e: **Chat_Inbox-You_Rekey-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Chat_Inbox-You_Rekey-ERROR-f84fce3eda1e.png)
 * Error in ccadec96c565: **Chat_Inbox-Part_Rekey_Selected-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Chat_Inbox-Part_Rekey_Selected-ERROR-ccadec96c565.png)
 * Error in f84fce3eda1e: **Chat_Inbox-Part_Rekey_Selected-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Chat_Inbox-Part_Rekey_Selected-ERROR-f84fce3eda1e.png)
 * Error in ccadec96c565: **Chat_Inbox-Part_Rekey-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Chat_Inbox-Part_Rekey-ERROR-ccadec96c565.png)
 * Error in f84fce3eda1e: **Chat_Inbox-Part_Rekey-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Chat_Inbox-Part_Rekey-ERROR-f84fce3eda1e.png)
 * Error in ccadec96c565: **Chat_Inbox-Long_Top-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Chat_Inbox-Long_Top-ERROR-ccadec96c565.png)
 * Error in f84fce3eda1e: **Chat_Inbox-Long_Top-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Chat_Inbox-Long_Top-ERROR-f84fce3eda1e.png)
 * Error in ccadec96c565: **Chat_Inbox-Long_Bottom-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Chat_Inbox-Long_Bottom-ERROR-ccadec96c565.png)
 * Error in f84fce3eda1e: **Chat_Inbox-Long_Bottom-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Chat_Inbox-Long_Bottom-ERROR-f84fce3eda1e.png)
 * Error in ccadec96c565: **Chat_Inbox-Empty-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Chat_Inbox-Empty-ERROR-ccadec96c565.png)
 * Error in f84fce3eda1e: **Chat_Inbox-Empty-ERROR** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Chat_Inbox-Empty-ERROR-f84fce3eda1e.png)
 * Added: **You_Rekey-Normal** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/You_Rekey-Normal-f84fce3eda1e.png)
 * Added: **Chat_Banner_Invite-Invite** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Chat_Banner_Invite-Invite-f84fce3eda1e.png)
 * Added: **Chat_Banner_Info-Info** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Chat_Banner_Info-Info-f84fce3eda1e.png)
 * Added: **Chat_Banner_Error-Error** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Chat_Banner_Error-Error-f84fce3eda1e.png)
 * Added: **Chat_Banner_Broken-Broken_Tracker_3** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Chat_Banner_Broken-Broken_Tracker_3-f84fce3eda1e.png)
 * Added: **Chat_Banner_Broken-Broken_Tracker_2** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Chat_Banner_Broken-Broken_Tracker_2-f84fce3eda1e.png)
 * Added: **Chat_Banner_Broken-Broken_Tracker_1** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Chat_Banner_Broken-Broken_Tracker_1-f84fce3eda1e.png)
 * Changed: **Text-Terminal_Empty_Information** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Text-Terminal_Empty_Information-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Text-Terminal_Empty_Information-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Text-Terminal_Empty_Information.png)
 * Changed: **Text-Terminal_Comment_Information** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Text-Terminal_Comment_Information-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Text-Terminal_Comment_Information-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Text-Terminal_Comment_Information.png)
 * Changed: **Text-Body_Wrap_Information** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Text-Body_Wrap_Information-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Text-Body_Wrap_Information-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Text-Body_Wrap_Information.png)
 * Changed: **Text-Body_Small_Semibold_Italic_Information** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Text-Body_Small_Semibold_Italic_Information-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Text-Body_Small_Semibold_Italic_Information-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Text-Body_Small_Semibold_Italic_Information.png)
 * Changed: **Text-Body_Small_Secondary_Link_Information** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Text-Body_Small_Secondary_Link_Information-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Text-Body_Small_Secondary_Link_Information-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Text-Body_Small_Secondary_Link_Information.png)
 * Changed: **Text-Body_Small_Primary_Link_Information** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Text-Body_Small_Primary_Link_Information-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Text-Body_Small_Primary_Link_Information-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Text-Body_Small_Primary_Link_Information.png)
 * Changed: **Text-Body_Big_Documentation** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Text-Body_Big_Documentation-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Text-Body_Big_Documentation-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Text-Body_Big_Documentation.png)
 * Changed: **Colors-white_20_rgba_255_255_255_0_20** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-white_20_rgba_255_255_255_0_20-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-white_20_rgba_255_255_255_0_20-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-white_20_rgba_255_255_255_0_20.png)
 * Changed: **Colors-white_0_rgba_255_255_255_0** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-white_0_rgba_255_255_255_0-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-white_0_rgba_255_255_255_0-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-white_0_rgba_255_255_255_0.png)
 * Changed: **Colors-orange_ff_6_f_21** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-orange_ff_6_f_21-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-orange_ff_6_f_21-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-orange_ff_6_f_21.png)
 * Changed: **Colors-grey_cccccc** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-grey_cccccc-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-grey_cccccc-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-grey_cccccc.png)
 * Changed: **Colors-green_3_E_8_FAF_3** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-green_3_E_8_FAF_3-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-green_3_E_8_FAF_3-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-green_3_E_8_FAF_3.png)
 * Changed: **Colors-dark_Blue_3_0_a_3052** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-dark_Blue_3_0_a_3052-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-dark_Blue_3_0_a_3052-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-dark_Blue_3_0_a_3052.png)
 * Changed: **Colors-blue_5_F_8_FAFD** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-blue_5_F_8_FAFD-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-blue_5_F_8_FAFD-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-blue_5_F_8_FAFD.png)
 * Changed: **Colors-blue_4_ebf_5_fc** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-blue_4_ebf_5_fc-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-blue_4_ebf_5_fc-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-blue_4_ebf_5_fc.png)
 * Changed: **Colors-blue_33_a_0_ff** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-blue_33_a_0_ff-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-blue_33_a_0_ff-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-blue_33_a_0_ff.png)
 * Changed: **Colors-black_10_rgba_0_0_0_0_10** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-black_10_rgba_0_0_0_0_10-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-black_10_rgba_0_0_0_0_10-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-black_10_rgba_0_0_0_0_10.png)
 * Changed: **Colors-black_05_rgba_0_0_0_0_05** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-black_05_rgba_0_0_0_0_05-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-black_05_rgba_0_0_0_0_05-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Colors-black_05_rgba_0_0_0_0_05.png)
 * Changed: **Checkbox-Disabled_unchecked** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Checkbox-Disabled_unchecked-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Checkbox-Disabled_unchecked-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Checkbox-Disabled_unchecked.png)
 * Changed: **Buttons-Unfollow** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Unfollow-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Unfollow-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Unfollow.png)
 * Changed: **Buttons-Secondary_waiting** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Secondary_waiting-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Secondary_waiting-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Secondary_waiting.png)
 * Changed: **Buttons-Secondary_full_Width_waiting** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Secondary_full_Width_waiting-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Secondary_full_Width_waiting-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Secondary_full_Width_waiting.png)
 * Changed: **Buttons-Primary_disabled** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Primary_disabled-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Primary_disabled-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Primary_disabled.png)
 * Changed: **Buttons-Follow_small_waiting** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Follow_small_waiting-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Follow_small_waiting-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Follow_small_waiting.png)
 * Changed: **Buttons-Danger_small_waiting** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Danger_small_waiting-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Danger_small_waiting-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Danger_small_waiting.png)
 * Changed: **Buttons-Danger_small** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Danger_small-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Danger_small-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Danger_small.png)
 * Changed: **Buttons-Danger_full_Width** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Danger_full_Width-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Danger_full_Width-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Danger_full_Width.png)
 * Changed: **Buttons-Danger** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Danger-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Danger-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Buttons-Danger.png)
 * Changed: **Avatar-Normal_x_80** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Normal_x_80-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Normal_x_80-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Normal_x_80.png)
 * Changed: **Avatar-Normal_x_32** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Normal_x_32-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Normal_x_32-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Normal_x_32.png)
 * Changed: **Avatar-Normal_x_24** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Normal_x_24-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Normal_x_24-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Normal_x_24.png)
 * Changed: **Avatar-Mutual_Follow_x_64** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Mutual_Follow_x_64-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Mutual_Follow_x_64-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Mutual_Follow_x_64.png)
 * Changed: **Avatar-Mutual_Follow_x_48** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Mutual_Follow_x_48-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Mutual_Follow_x_48-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Mutual_Follow_x_48.png)
 * Changed: **Avatar-Follows_You_x_80** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Follows_You_x_80-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Follows_You_x_80-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Follows_You_x_80.png)
 * Changed: **Avatar-Follows_You_x_16** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Follows_You_x_16-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Follows_You_x_16-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Follows_You_x_16.png)
 * Changed: **Avatar-Follows_You_x_112** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Follows_You_x_112-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Follows_You_x_112-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Follows_You_x_112.png)
 * Changed: **Avatar-Following_x_32** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Following_x_32-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Following_x_32-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Following_x_32.png)
 * Changed: **Avatar-Following_x_176** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Following_x_176-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Following_x_176-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Following_x_176.png)
 * Changed: **Avatar-Fallback_x_48** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Fallback_x_48-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Fallback_x_48-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Fallback_x_48.png)
 * Changed: **Avatar-Fallback_x_16** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Fallback_x_16-ccadec96c565.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Fallback_x_16-f84fce3eda1e.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1507833249976-ccadec96c565-f84fce3eda1e-linux/Avatar-Fallback_x_16.png)
</summary>

**Pull request closed without merging**